pkg/controller/operatingsystemconfig/actuator_test.go

```diff
@@ -55,7 +55,7 @@ var _ = Describe("Actuator", func() {
 					Type: gardenlinux.OSTypeGardenLinux,
 				},
 				Purpose: extensionsv1alpha1.OperatingSystemConfigPurposeProvision,
-				Units:   []extensionsv1alpha1.Unit{{Name: "some-unit", Content: pointer.String("foo")}},
+				Units:   []extensionsv1alpha1.Unit{{Name: "some-unit", Content: ptr.To[string]("foo")}},
```

Comment:
I think
```suggestion
				Units:   []extensionsv1alpha1.Unit{{Name: "some-unit", Content: ptr.To("foo")}},
```
should work.